doc-content/enterprise-only/project-data/preview-editor-data-objects-import-proc.adoc

```diff
@@ -0,0 +1,16 @@
+[id='preview-editor-data-objects-import-proc']
+= Importing data objects
+
+*Test Scenario (Preview)* editor will load all data objects that are located in the same package as the test scenario. You can view all the data objects from the *Data Objects* tab in the editor. The loaded data objects are also displayed in the *Test Editor* tab in the *Test Tools* panel.
```

Comment:
Avoid "will".
Suggestion: *Test Scenario (Preview)* editor loads all data objects that...."

Reply:
Changed as suggested.